test/shared/src/main/scala/zio/test/Assertion.scala

```diff
@@ -282,13 +282,94 @@ object Assertion extends AssertionVariants {
     }
 
   /**
-   * Makes a new assertion that requires an iterable contain the specified
-   * element. See [[Assertion.exists]] if you want to require an iterable to contain an element
+   * Makes a new assertion that requires a numeric value is positive.
+   */
+  def isPositive[A](implicit num: Numeric[A]): Assertion[A] =
+    Assertion.assertion("isPositive")()(actual => num.gt(actual, num.zero))
+
+  /**
+   * Makes a new assertion that requires a numeric value is non negative.
+   */
+  def nonNegative[A](implicit num: Numeric[A]): Assertion[A] =
```

Comment:
I initially wrote these numeric assertions in this way. I changed them so that the name in the error message could be more specific. If this is not a concern, I will go with this approach.

Reply:
One thing we could explore here is adding a `relabel` combinator that lets you replace the existing label with a new one. Then we could implement these assertions in terms of other ones and still have the more specific names.

Reply:
I like the idea to introduce a new combination to allow the label to be replaced. I will experiment with this idea.